### DISCUSSION TITLE
Check if crashlytics is enabled (v5)

### DISCUSSION THREAD

**Discussion by ajfick** [invertase/react-native-firebase#4256] [Q&A] [answered]
Is it possible to check if Crashlytics reporting is enabled in v5? 
I'm manually initializing it on the release builds only using the [this documentation.](https://v5.rnfirebase.io/docs/v5.x.x/crashlytics/manual-initialization)and the following snippet in my index.js file: 
```js
if (!__DEV__) {
  firebase.crashlytics().enableCrashlyticsCollection();
}
```

My app has an issue reporting page that I'd like to be able to show if error reporting is enabled on. Is there any way to check that so that I can display to the user that crash reports are on?

I know that I could probably just show it based on `__DEV__` but, I feel like it's safer to do it based on `firebase.crashlytics()` instead in case it fails to initialize or something.

**Comment by mikehardy** [comment, marked as answer]
v5 is well out of support, and you are up against a migration deadline for crashlytics, the new version of which is only available on current stable versions: https://firebase.google.com/docs/crashlytics/get-started?platform=ios

> Note: The Fabric SDK is now deprecated and will continue reporting your app's crashes until November 15, 2020. On this date, the Fabric SDK and old versions of the Firebase Crashlytics SDK will stop sending crashes for your app. To continue getting crash reports in the Firebase console, make sure you upgrade to the Firebase Crashlytics SDK versions 17.0.0+ for Android, 4.0.0+ for iOS, and 6.15.0+ for Unity.

migrate to current stable versions, there are APIs that do what you want https://rnfirebase.io/reference/crashlytics#isCrashlyticsCollectionEnabled

**Reply by ajfick** [reply]
Will do! Thanks for the quick reply!
